crates/core/tedge/src/cli/mqtt/cli.rs

```diff
@@ -45,6 +46,12 @@ pub enum TEdgeMqttCli {
         /// Avoid printing the message topics on the console
         #[clap(long = "no-topic")]
         hide_topic: bool,
+        /// Set a timeout duration (e.g., 60s, 1h). Use 0 for no timeout
+        #[clap(long, default_value = "0")]
+        window: SecondsOrHumanTime,
+        /// Set the number of packets before stopping. Use 0 for unlimited
+        #[clap(long, default_value = "0")]
+        packets: u32,
```

Comment:
A cleaner approach is to use optional inputs instead of magic values.

```suggestion
        /// Set a timeout duration (e.g., 60s, 1h).
        #[clap(long)]
        window: Option<SecondsOrHumanTime>,
        /// Set the number of packets before stopping.
        #[clap(long)]
        packets: Option<u32>,
```

Reply:
Done in 20884905d1e22b2e6db220a89fc9b773abd8c334